app/src/main/java/com/kovacnet/waterlevel/StatsActivity.java

```diff
@@ -163,6 +163,31 @@ private void fillStats()
 
 
     }
+    class Motor{  
+    	
+    	motor1(){System.out.println("emtying and filling motors");}  
```

Comment:
Are you sure this is working correctly in android app ?
Looks like a console application to me.